Formula/postgis.rb

```diff
@@ -48,6 +48,7 @@ def postgresql
 
   def install
     ENV.deparallelize
+    ENV.append "CXXFLAGS", "-std=c++14" # Needed for GDAL 3.7+
```

Comment:
We need C++17, it seems.

Reply:
Addressed in ca731c61009511f35e98b7def8db09dff43a4ac2